### DISCUSSION TITLE
文档有点少啊.如何进行安装?现在就看到文档有个安装agent 这就kindling的全部了么?

### DISCUSSION THREAD

**Discussion by jokerZhou** [harmonycloud/kindling#614] [agent collector]
文档有点少啊.如何进行安装?现在就看到文档有个安装agent 这就kindling的全部了么?

**Comment by dxsup** [comment]
是的，Kindling是比较轻量的，安装agent就能采集数据了，[查看安装文档](https://kindlingx.com/docs/installation/kindling-agent/install-kindling-in-kubernetes/).
要查看数据可以直接使用prometheus，[查看指标文档](https://kindlingx.com/docs/usage/prometheus-metric/)。
还提供一个Grafana插件用来查看拓扑图，[查看安装文档](https://kindlingx.com/docs/installation/setting-up-grafana/).

**Reply by jokerZhou** [reply]
https://kindlingx.com/cn/docs/usage/trace-profiling-manual/ 但是我没看到这个文档的控制台在哪里= =

**Reply by dxsup** [reply]
看这个文档 https://kindlingx.com/docs/usage/enable-trace-profiling/

**Reply by jokerZhou** [reply]
哦哦 好的  谢谢 没仔细看...而且这个端口是直接开的么?没在svc上看到..

**Reply by dxsup** [reply]
使用了hostNetwork

**Reply by jokerZhou** [reply]
OK  十分感谢 看到了 ... 可以继续愉快的研究了
